ethstore/src/json/kdf.rs

```diff
@@ -111,7 +111,7 @@ pub struct Pbkdf2 {
 	pub c: u32,
 	pub dklen: u32,
 	pub prf: Prf,
-	pub salt: H256,
+	pub salt: Bytes,
```

Comment:
why is it changed to `Bytes`?

Reply:
again, it's bc we do not know length of salt